core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -125,6 +130,22 @@ class SslAdminIntegrationTest extends SaslSslAdminIntegrationTest {
     startSasl(jaasSections(List.empty, None, ZkSasl))
   }
 
+  override def createConfig: util.Map[String, Object] = {
+    val config = new util.HashMap[String, Object]
+    config.put(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, "20000")
+    config.put(AdminClientConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, "40000")
+    config
+  }
+
+  override def modifyControllerConfigs(configs: Seq[Properties]): Unit = {
+    JaasTestUtils.sslConfigs(ConnectionMode.SERVER, false, OptionConverters.toJava(trustStoreFile), s"controller")
+      .forEach((k, v) => configs.foreach(c => c.put(k, v)))
+  }
+
+  override def extraControllerSecurityProtocols(): Seq[SecurityProtocol] = {
```

Comment:
ditto. maybe it can override `newKRaftQuorum`?

Reply:
Sadly the listener related configs are hard code inside `protected def newKRaftQuorum(overridingProps: Properties): KRaftQuorumImplementation`, so I have to add another method to override those configs.